docs/get-started/clinc150/index.md

```diff
@@ -0,0 +1,843 @@
+For more info on the CLINC150 dataset, check out the
+[dataset repo](https://github.com/clinc/oos-eval).
+
+Firstly, let's clone the dataset locally:
+```bash
+git clone https://github.com/clinc/oos-eval.git
+```
+
+CLINC150 comes in different sizes with regards to the number of utterances per intent,
+to facilitate experimentation on few-shot learning methods. We will use the full version
+which should be located in `./oos-eval-master/data/data_full.json`.
+
+The dataset is a JSON file with utterance intent pairs. We convert the train, val and
+test split to `DocumentArray`s and attach the intent label for each doc in 
+`doc.tags['finetuner_label']`.
+
+```python
+import json
+
+from docarray import Document, DocumentArray
+
+DATASET_PATH = 'oos-eval-master/data/data_full.json'
+
+with open(DATASET_PATH, 'r') as f:
+    data = json.load(f)
+
+train_data = DocumentArray(
+    [
+        Document(text=utterance, tags={'finetuner_label': intent})
+        for utterance, intent in data['train']
+    ]
+)
+val_data = DocumentArray(
+    [
+        Document(text=utterance, tags={'finetuner_label': intent})
+        for utterance, intent in data['val']
+    ]
+)
+test_data = DocumentArray(
+    [
+        Document(text=utterance, tags={'finetuner_label': intent})
+        for utterance, intent in data['test']
+    ]
+)
+
+print(f'Num train samples: {len(train_data)}')
+print(f'Num val samples: {len(val_data)}')
+print(f'Num test samples: {len(test_data)}')
+```
+```
+Num train samples: 15000
+Num val samples: 3000
+Num test samples: 4500
+```
+
+Let's investigate the intent distribution. We use `collections.Counter` to compute
+the distribution of our train, val and test intents as well as the average number of
+utterances per intent in all 3 splits.
+
+```python
+from collections import Counter
+
+train_intents = Counter([doc.tags['finetuner_label'] for doc in train_data]).most_common()
+val_intents = Counter([doc.tags['finetuner_label'] for doc in val_data]).most_common()
+test_intents = Counter([doc.tags['finetuner_label'] for doc in test_data]).most_common()
+
+num_utts_per_intent_train = sum([v for _, v in train_intents]) / len(train_intents)
+num_utts_per_intent_val = sum([v for _, v in val_intents]) / len(val_intents)
+num_utts_per_intent_test = sum([v for _, v in test_intents]) / len(test_intents)
+
+print(f'Num train intents: {len(train_intents)}')
+print(f'Num val intents: {len(val_intents)}')
+print(f'Num test intents: {len(test_intents)}')
+
+print(f'Avg num utterances per intent in train set: {num_utts_per_intent_train}')
+print(f'Avg num utterances per intent in val set: {num_utts_per_intent_val}')
+print(f'Avg num utterances per intent in test set: {num_utts_per_intent_test}')
+```
+```
+Num train intents: 150
+Num val intents: 150
+Num test intents: 150
+Avg num utterances per intent in train set: 100.0
+Avg num utterances per intent in val set: 20.0
+Avg num utterances per intent in test set: 30.0
+```
```

Comment:
As this tutorial is quite long, I would omit this code part entirely, and just state these numbers in text - especially since they are "fixed" and already mentioned at the beginning of dataset readme

Reply:
there are variables in this code block that are used afterwards

Reply:
But those variables are already known (stated in readme of dataset)